client/my-sites/stats/stats-subscribers-chart-section/index.tsx

```diff
@@ -77,7 +77,7 @@ export default function SubscribersChartSection( {
 		siteId,
 		period,
 		quantity,
-		queryDate
+		queryDate.toISOString()
```

Comment:
Let's avoid changing the parameter when executing a function.
Can you keep this as a `Date` object? This way, we avoid converting a date to a string, a date and a string again.

Reply:
If a string is needed, convert it inside `useSubscribersQuery` instead of the function execution here.